packages/fxa-auth-client/lib/crypto.ts

```diff
@@ -222,7 +222,7 @@ export async function jweDecrypt(
   jwe: string
 ): Promise<string> {
   const encoder = new TextEncoder();
-  const [header, empty, iv, ciphertext, authenticationTag] = jwe.split('.');
+  const [header, , iv, ciphertext, authenticationTag] = jwe.split('.');
```

Comment:
Interesting that earlier I saw `undefined` being added but here we just leave it empty.  Linter need to make up its mind.

Reply:
> _The next step will be to start arguing about what rules to enforce 😛 and applying them consistently._

"semicolon has entered the chat"